modules/pam_rootok/pam_rootok.c

```diff
@@ -66,6 +66,7 @@ log_callback (int type UNUSED, const char *fmt, ...)
 	ret = vasprintf (&buf, fmt, ap);
 	va_end(ap);
 	if (ret < 0) {
+        close(audit_fd);
```

Comment:
Let's use `audit_close` here.
Also please watch the indentation.

Reply:
Done, and sorry for the indentation, I wasn't aware of it as usually the IDE handles it correctly